yarn.lock

```diff
@@ -1955,19 +2052,56 @@
     "@file-services/path" "^7.0.1"
     "@file-services/types" "^7.0.1"
 
+"@wixc3/common@^3.0.4":
+  version "3.0.4"
+  resolved "https://registry.yarnpkg.com/@wixc3/common/-/common-3.0.4.tgz#12285ace098a677e73da14c1a47bdf793bd3ebc0"
+  integrity sha512-qhzsY9482lkFxUu6zamfggtHI44PF2u2e+QQTgLRZ6oBMpudHmdswd5jGuN7H0l4Ohxlgz4I0eSy8uotVKu2mA==
+  dependencies:
+    "@file-services/path" "^7.0.1"
+    "@file-services/types" "^7.0.1"
+
 "@wixc3/create-disposables@^2.2.0":
   version "2.2.0"
   resolved "https://registry.yarnpkg.com/@wixc3/create-disposables/-/create-disposables-2.2.0.tgz#848b730751121ceceaab0ab0c28ebf550f1eb1ef"
   integrity sha512-YQAvDKG8k503Pnak6kj0lNbGaLcyZSjFipFmK17rcE2jpLOrA5B72jrUAPGCbDrXnVCyc+U7Xmr8eyxxKRoIuQ==
 
-"@wixc3/fs-utils@^3.0.3":
+"@wixc3/docs@^3.0.4":
+  version "3.0.4"
+  resolved "https://registry.yarnpkg.com/@wixc3/docs/-/docs-3.0.4.tgz#e94d871702fdb407191449ffb60e9cca5390417b"
+  integrity sha512-YA5QK4OYL+8Xgg21IXpTIcLDF4Vww+OJeXVjGpT5g+JfyskO1ZFOe7RY4k1gk4LGvh+pqfg3X5euNWsllI9p2Q==
+  dependencies:
+    "@microsoft/api-documenter" "^7.19.24"
+    "@microsoft/api-extractor" "^7.33.6"
+    "@microsoft/api-extractor-model" "^7.25.2"
+    "@wixc3/common" "^3.0.4"
+    "@wixc3/fs-utils" "^3.0.4"
+    prettier "^2.7.1"
```

Comment:
IMO, our docs generator shouldn't do code formatting. mixing the two concerns is not required.

Reply:
we discussed offline and agreed to keep it for generated MD formatting